smdebug/tensorflow/base_hook.py

```diff
@@ -437,7 +437,7 @@ def _write_for_tensor(self, tensor_name, tensor_value, save_collections, tensor_
 
         # this tensor_name is tf tensor name, need to convert to export_name
         tensor_ref = self._get_tensor_ref(tensor_name, save_collections=save_collections)
-        if tensor_ref:
+        if tensor_ref is not None:
```

Comment:
why dint the original if condition work?

Reply:
`if obj` executes `obj.__nonzero__()`
where `if obj is not None:` checks if it is not None.

This is  a nit fix.